lib/md043.js

```diff
@@ -24,17 +25,23 @@ module.exports = {
       forEachHeading(params, (heading, content) => {
         if (!hasError) {
           anyHeadings = true;
-          const actual = levels[heading.tag] + " " + content;
-          const expected = getExpected();
+          const actual = matchCase ?
+            levels[heading.tag] + " " + content :
+            levels[heading.tag] + " " + content.toLowerCase();
```

Comment:
I'd like to avoid the duplication that's been added. What about adding a helper like `const handleCase = (s) => match_case ? s : s.toLowerCase()`? Then wrap `content`, etc. with that.